src/tabmat/ext/sparse.pyx

```diff
@@ -35,7 +35,7 @@ def sparse_sandwich(A, AT, floating[:] d, win_integral[:] rows, win_integral[:]
 
     cdef floating* dp = &d[0]
 
-    cdef win_integral m = cols.shape[0]
+    cdef long long m = cols.shape[0]
```

Comment:
Please use the integer types with explicit size. You will need to import `from libc.stdint cimport int64_t` at the top.

```suggestion
    cdef int64_t m = cols.shape[0]
```

Reply:
Should this be changed elsewhere as well? Currently, we use `win_integral` (fused type of integral and long long) very liberally throughout (see https://github.com/Quantco/tabmat/pull/87).

Reply:
Yes, it would make more sense if the fused type would also use only sized integers. I guess though that here `int64_t` is the right one always.